src/wp-includes/class-wp-duotone.php

```diff
@@ -1300,4 +1300,45 @@ public static function get_filter_css_property_value_from_preset( $preset ) {
 
 		return 'url(#' . $filter_id . ')';
 	}
+
+	/**
+	 * Fixes the issue with our generated class name not being added to the block's outer container
+	 * in classic themes.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param  string $block_content Rendered block content.
+	 * @return string                Filtered block content.
```

Comment:
@tellthemachines `@return string Filtered block content.` space should be removed.

Reply:
Fixed! https://core.trac.wordpress.org/changeset/58316